src/oge/data_cleaning.py

```diff
@@ -2145,3 +2146,349 @@ def aggregate_cems_to_subplant(cems):
     cems = apply_dtypes(cems)
 
     return cems
+
+
+class AnomalyScreeningFirstStep:
+    """Screen timeseries for anomalous value following the algorithm steps described
+    in Tyler H. Ruggles et al. Developing reliable hourly electricity demand data
+    through screening and imputation (2020)
+
+    Note that the screening algorithms have been developed for demand timeseries and
+    some of these algorithms might not be tailored to other time series.
+
+    The screening is conducted in 2 steps. Step 1 removes the most egregious anomalies
+    where few or no calculations are needed. Afterward, in Step 2, the most extreme
+    values have been removed making calculations of local characteristics of the data
+    more reasonable. Through this screening process hourly values can be recategorized
+    from okay to other classifications based on the algorithms.
+    """
+
+    def __init__(
+        self, df: pd.DataFrame, field: str, **kwargs: Optional[dict[str, Any]]
+    ):
+        """Constructor
+
+        Arguments
+        ---------
+        * `df` : timeseries as a data frame.
+        * `field` : Column to screen.
+        * `kwargs` : Optional keyword arguments.
+        """
+        self.df = df.copy().reset_index()
+        self.field = field
+        self.global_cut_multiplier = (
+            kwargs["global_cut_multiplier"] if "global_cut_multiplier" in kwargs else 10
```

Comment:
is this the default multiplier (10x the median) for identifying extremes? We may want to experiment with this to see what an appropriate multiplier for generation data is. A smaller multiplier might be appropriate.

Reply:
Yes x10 is the default multiplier. I agree that we should fine tune this value that's why it can be change via `kwargs`.